spec/index.html

```diff
@@ -2627,6 +2627,46 @@ <h3>Algorithm</h3>
   </section>
 </section>
 
+<section id="serialization">
+  <h2>Serialization</h2>
+
+  <p>This section describes the process of creating a serialized [[N-Quads]] representation
+    of a <a>normalized dataset</a>.</p>
+
+  <p>The <dfn class="lint-ignore">serialized canonical form</dfn> of a <a>normalized dataset</a>
+    is an <a data-cite="N-QUADS#n-quads-document">N-Quads document</a>
+    created by representing each <a>quad</a> from the <a>normalized dataset</a>
+    in <a>canonical n-quads form</a>,
+    sorting them into <a>code point order</a>,
+    and concatenating them.
+    The resulting document document has a media type of `application/n-quads`,
+    as described in <a data-cite="N-QUADS#sec-mediaReg">C. N-Quads Internet Media Type, File Extension and Macintosh File Type</a>
+    of [[N-QUADS]].</p>
+
+  <p>When serializing quads in <a>canonical n-quads form</a>
+    components which are <a>blank nodes</a> MUST be serialized using the
+    canonical label associated with the <a>blank node</a> in the <a>normalized dataset</a>.</p>
+
+  <aside id="ex-ser-unique-hashes"
+        class="example"
+        title="Canonical N-Quads representation of Unique Hashes">
+
+    <p>This example illustrates the result of serializing
+      the normalized dataset described in <a href="#ex-ca-unique-hashes"></a>.</p>
+
+    <pre id="ex-ser-unique-hashes-result"
+         data-transform="updateExample"
+         title="Canonical N-Quads representation of Unique Hashes">
+    <!--
+      <http://example.com/p> <http://example.com/q> _:c14n0 .
```

Comment:
Some parts after `//`s are wrongly misunderstood as JavaScript comments and unexpectedly indented. Could we specify this code example is not JavaScript to avoid this?

Reply:
I added `class="nohighlight"` to the `pre` element, which looks like it does it.